yearn/apy/v2.py

```diff
@@ -68,9 +67,25 @@ def simple(vault, samples: ApySamples) -> Apy:
     apys = [week_ago_apy, month_ago_apy, inception_apy]
     net_apy = next((value for value in apys if value != 0), 0)
 
-    # performance fee is doubled since 1x strategists + 1x treasury
-    performance = (contract.performanceFee() * 2) if hasattr(contract, "performanceFee") else 0
+    # for performance fee, half comes from strategy (strategist share) and half from the vault (treasury share)
+    strategy_fees = []
+    for x in range[20]: # we can have up to 20 strategies
```

Comment:
```suggestion
    for x in range(0, 19): # we can have up to 20 strategies
```

Reply:
the correct way for [0-19] would be `range(20)`

Reply:
ty ser, now that I'm able to better test this stuff after discussing with jstash should be far fewer dumb errors like this